spec/dmv_spec.rb

```diff
@@ -32,12 +36,66 @@
       @dmv.add_facility(@facility_2)
       @dmv.add_facility(@facility_3)
-
+      
       expect(@dmv.facilities_offering_service('Road Test')).to eq([@facility_2, @facility_3])
     end
   end
+  
+  describe '#create_facility_helper' do
+    it 'converts co facility information from API correct format' do
+      co_facility_info = @dmv.create_facility_helper(@co_dmv_office_locations)
+      expect(co_facility_info.class).to eq(Array)
+      expect(co_facility_info.first).to eq({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+    end
+    
+    it 'converts ny facility information from API correct format' do
+      ny_facility_info = @dmv.create_facility_helper(@ny_dmv_office_locations)
+      expect(ny_facility_info.class).to eq(Array)
+      expect(ny_facility_info.first).to eq({name: 'EVANS COUNTY OFFICE', address: '6853 ERIE RD DERBY NY 14006', phone: '(716) 858-7450'})
+    end
+
+    it 'converts MO facility information from API correct format' do
+      mo_facility_info = @dmv.create_facility_helper(@mo_dmv_office_locations)
+      expect(mo_facility_info.class).to eq(Array)
+      expect(mo_facility_info.first).to eq({name: 'OAKVILLE', address: '3164 TELEGRAPH ROAD ST LOUIS MO 63125', phone: '(314) 887-1050'})
+    end
+  end
+
+  describe '#create_facility' do
+    it 'converts co facility information from API' do
+      @dmv.create_facility_helper(@co_dmv_office_locations)
+      expect(@dmv.create_facility.count).to eq(5)
+      expect(@dmv.create_facility.class).to eq(Array)
+      expect(@dmv.create_facility.first.class).to eq(Facility)
+      expect(@dmv.create_facility.first.name).to eq('DMV Tremont Branch')
+      expect(@dmv.create_facility.first.phone).to eq('(720) 865-4600')
+      expect(@dmv.create_facility.first.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+    end
+
+    it 'converts ny facility information from API' do
+      @dmv.create_facility_helper(@ny_dmv_office_locations)
+      expect(@dmv.create_facility.count).to eq(172)
+      expect(@dmv.create_facility.class).to eq(Array)
+      expect(@dmv.create_facility.first.class).to eq(Facility)
+      expect(@dmv.create_facility.first.name).to eq('EVANS COUNTY OFFICE')
+      expect(@dmv.create_facility.first.phone).to eq('(716) 858-7450')
+      expect(@dmv.create_facility.first.address).to eq('6853 ERIE RD DERBY NY 14006')
+    end
+    
+    it 'converts mo facility information from API' do
+      @dmv.create_facility_helper(@mo_dmv_office_locations)
+      expect(@dmv.create_facility.count).to eq(178)
+      expect(@dmv.create_facility.class).to eq(Array)
+      expect(@dmv.create_facility.first.class).to eq(Facility)
+      expect(@dmv.create_facility.first.name).to eq('OAKVILLE')
+      expect(@dmv.create_facility.first.phone).to eq('(314) 887-1050')
+      expect(@dmv.create_facility.first.address).to eq('3164 TELEGRAPH ROAD ST LOUIS MO 63125')
+    end
+  end
```

Comment:
Again, great tests, and you could also make them more robust by iterating within the test examples.
```
@dmv.create_facility_helper(@mo_dmv_office_locations)
facility_objects = @dmv.create_facility
facility_objects.each do |facility|
  expect(facility).to be_a(Facility)
  expect(facility.name).to be_a(String)
end
```
Alternatively, you could assert the specific attributes of `@dmv.create_facility.last` in addition to the first ones.